OracleJava/22/Dockerfile

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2022, 2024 Oracle and/or its affiliates.
```

Comment:
Comma needed after "2024"

Reply:
I'm seeing that this comma isn't present in all of the files. Perhaps I'm wrong and the expected format of the copyright statement has changed. @Djelibeybi, do you know?

Reply:
The authoritive internal example has the comma but I've honestly seen it provided with and without one from our various legal counsels over the years, so I guess it doesn't really matter. I personally _hate_ that second comma (for no rational reason, I admit) so I'm perfectly fine without it. ;)

Reply:
Give @Djelibeybi's answer.  Is there any further changes required for this pull request or can it proceed as is?

Reply:
Based on Avi's answer, let's go as is. I'm just thinking about all of the grief I've given my teams about these commas.